resources/stylesheets/product.css

```diff
@@ -0,0 +1,180 @@
+.product-List .flex-container {
+  flex-wrap: wrap; }
+
+.product-List__filter {
+  background-color: #eaeaea;
+  width: 100%;
+  padding-left: 0;
+  margin-top: 30px;
+  margin-bottom: 30px; }
+  .product-List__filter input {
+    display: none; }
+  .product-List__filter label {
+    display: block;
+    cursor: pointer;
+    padding: 13px 15px;
+    font-size: 17px;
+    position: relative; }
+    .product-List__filter label:after {
+      content: "";
+      position: absolute;
+      right: 18px;
+      top: 12px;
+      height: 14px;
+      width: 14px;
+      border-left: 2px solid #fff;
+      transform: rotate(-45deg);
+      border-bottom: 2px solid #fff; }
+  .product-List__filter .dropdown-content {
+    width: 100%;
+    height: 0px;
+    overflow: hidden;
+    list-style-type: none;
+    transition: height 0.5s ease-out;
+    display: block; }
+    .product-List__filter .dropdown-content li {
```

Comment:
Why are you mixing BEM and non-BEM naming convention?

Reply:
Here dropdown-content is working as a Block element